src/usr/local/www/services_ntpd.php

```diff
@@ -93,6 +93,11 @@
 	}
 
 	if (!$input_errors) {
+		if (isset($pconfig['enable'])) {
+			$config['ntpd']['enable'] = 'enabled';
+		} else {
+			$config['ntpd']['enable'] = 'disabled';
+		}
```

Comment:
This could be simplified:

`$config['ntpd']['enable'] = isset($pconfig['enable']) ? 'enabled' : 'disabled';`

Reply:
thanks,
all fixed